test.nix

```diff
@@ -170,6 +170,43 @@ let
         { buildInputs = [ dummyfication ]; }
         "my-bin > $out";
 
+      git-symlink =
+        let
+          dep = pkgs.runCommand "dep" {
+            buildInputs = [ pkgs.git ];
+          } ''
+            mkdir $out
+            cd $out
+            cp -ar ${./test/git-symlink/dep-workspace}/* .
+
+            git init
+            git add .
+            git config user.email 'someone'
+            git config user.name 'someone'
+            git commit -am 'Initial commit'
+          '';
+
+          app = pkgs.runCommand "app" {
+            buildInputs = [ pkgs.git ];
+          } ''
+            mkdir $out
+            cd $out
+            cp -ar ${./test/git-symlink/app}/* .
+
+            depPath="${dep}"
```

Comment:
This part is the same as for `git-dep-by-branch-with-slash` - for simplicity, I'm planning on refactoring it (i.e. extracting to some common function) in a separate commit later, after this PR.